internal/util/get/get.go

```diff
@@ -258,30 +258,30 @@ func (c *Command) DefaultValues() error {
 
 // upsertKptfile populates the KptFile values, merging any cloned KptFile and the
 // cloneFrom values.
-func (c *Command) upsertKptfile(spec *git.RepoSpec) error {
+func upsertKptfile(path, name string, spec *git.RepoSpec) error {
```

Comment:
Currently, we pass around a path of type `string` everywhere in the codebase. It's ambiguous whether it's absolute, relative, cleaned, .... Does it work for windows (I'm pretty sure it wouldn't work in general)? Does it work when there are symlinks? etc.

We should move towards making this an explicit type, so the compiler helps us. Relevant:

https://github.com/GoogleContainerTools/kpt/pull/1340/files#diff-f440b5f78da4e52ed45c778be5d7b49373a9427073d651b6166eb5bbd93b5d6a

Reply:
I have been working on making sure all paths passed around is always absolute, and any translation from relative to absolute happens in the cobra commands. Agree that we can consider a custom type for an absolute path.

Reply:
Depending on the usage, we can pass around:
1. Pkg: https://github.com/GoogleContainerTools/kpt/blob/next/internal/pkg/pkg.go#L35
2. UniquePath: https://github.com/GoogleContainerTools/kpt/blob/next/internal/pkg/pkg.go#L27